cookbooks/aws-parallelcluster-environment/templates/imds/parallelcluster-iptables.service.erb

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Setup ephemeral drives service
```

Comment:
This service is not about setting up ephemeral drivers, but about configuring iptables.

Reply:
Done